core/types/utxo.go

```diff
@@ -471,3 +476,7 @@ func UTXOHash(txHash common.Hash, index uint16, utxo *UtxoEntry) common.Hash {
 	binary.BigEndian.PutUint16(indexBytes, index)
 	return RlpHash([]interface{}{txHash, indexBytes, utxo}) // TODO: Consider encoding to protobuf instead
 }
+
+func CoinbaseLockupHash(ownerContract common.Address, beneficiaryMiner common.Address, delegate common.Address, lockupByte byte, epoch uint32, balance *big.Int, unlockHeight uint32, elements uint16) common.Hash {
+	return RlpHash([]interface{}{ownerContract, beneficiaryMiner, delegate, lockupByte, epoch, balance, unlockHeight, elements}) // TODO: Consider encoding to protobuf instead
```

Comment:
just to make sure i understand, what is elements here?

Reply:
Number of coinbases that make up this tranche